internal/expr/complete.go

```diff
@@ -41,6 +41,14 @@ func (pe *PreparedExpr) Complete(args ...any) (ce *CompletedExpr, err error) {
 		}
 		v := reflect.ValueOf(arg)
 		t := v.Type()
+
+		if t.Kind() != reflect.Struct {
+			if t.Kind() != reflect.Pointer {
```

Comment:
This won't work when the arg is a poitner to a struct.

We need something similar to `if v.Kind() == reflect.Pointer { v = v.Elem() }` beforehand, and then we don't need to worry about Pointers in the test here.